pypeman/remoteadmin.py

```diff
@@ -56,7 +56,7 @@ def get_channel(self, name):
         return channel by is name.all_channels
         """
         for chan in channels.all_channels:
-            if chan.name == name:
+            if chan.name == name or chan.short_name == name:
```

Comment:
I wouldn't do it here but just for fun / info.

Following would work as well.

`if name in (chan.name, chan.short_name)`

Reply:
ui, changed (everywhere)